docs/configuration_file.rst

```diff
@@ -1,22 +1,1167 @@
+|          stddev:                                     |                                  |                                                          |
+|          distribution:                               |                                  |                                                          |
+|        bulkvolume_mult_parameter_from_sim_model:     |                                  |                                                          |
++------------------------------------------------------+----------------------------------+----------------------------------------------------------+
+        
+
+relative_permeability
+---------------------
+
+FlowNet currently uses Corey correlations for generating relative permeability input curves for Flow. At a later 
+stage LET parametrization may also be implemented.
+
+
+scheme
+~~~~~~
+
+The scheme parameter decides how many sets of relative permeability curves to generate as
+input to Flow. There are three options. With **scheme: global** only one set of relative 
+permeability curves will be generated and applied to all flow tubes in the model. With
+**scheme: individual** all flow tubes in the model will have its own set of relative permeability
+curves. With **scheme: regions_from_sim** FlowNet will extract the SATNUM regions from the 
+input model provided, and assign the same set of relative permeability curves to all flow tubes 
+that are (mostly) located within the same SATNUM region. The default value is global.
+
+interpolate
+~~~~~~~~~~~
+
+SCAL experts will often provide three sets of relative permeability curves (one pessimistic set, 
+one base set and one optimistic set) to run sensitivities on a reservoir model. 
+This introduces the option of generating new sets of relative permeability curves within the 
+envelope created by the low/base/high sets of curves by using an interpolation parameter 
+(potentially two interpolation parameters in three phase models). This will limit the number of 
+history matching parameters, especially when the number of SATNUM regions is large. The default 
+value is False. 
+
+When using the interpolation option for relative permeability, some of the keywords related to choice 
+of `prior`_ distribution have a slightly different meaning. This applies to **min**, **base**, and **max**. 
+There is also an additional keyword **low_optimistic** which only is meaningful to define for relative permeability.
+
+Each of the input parameters needs a low, base, and high value to be defined. This is done through
+the **min** (low), **base** and **max** (high) keywords. 
+For some parameters a low numerical value is favorable. For example, a low value for *sorw* will provide
+a more optimistic relative permeability curve. This can be indicated by setting 
+**low_optimistic** to **True** for that parameter.
+
+A parameter value on the interval [-1,0) will interpolate all input parameters 
```

Comment:
I'm a bit confused but it might just be because I don't know enough about the topic. Does the value of the interpolation parameter make a difference or would any number in e.g. [-1,0) have the same effect?

Reply:
It's a history matching parameter, -1 will reproduce the low case, 0 will reproduce the base case, 1 will reproduce the high case. All other numbers will be linear interpolations between the inputs in the three defined cases.